scripts/linux/nv/gitlab_test_performance.sh

```diff
@@ -32,20 +32,26 @@ do
         set +e
         curl -s http://localhost:3000
         RC=$?
-        sleep 0.15
+        sleep 0.2
         set -e
 done
 echo ">>> Local Server started for further performance test ... >>>"
 
 export cache_location="cache"
 
 echo ">>> Start performance tests ... >>>"
-heaptrack ./build/tests/performance/olp-cpp-sdk-performance-tests --gtest_filter="*short_test_null_cache" 2>> errors.txt || TEST_FAILURE=1
-mv heaptrack.olp-cpp-sdk-performance-tests.*.gz short_test_null_cache.gz
-heaptrack ./build/tests/performance/olp-cpp-sdk-performance-tests --gtest_filter="*short_test_memory_cache" 2>> errors.txt || TEST_FAILURE=1
-mv heaptrack.olp-cpp-sdk-performance-tests.*.gz short_test_memory_cache.gz
-heaptrack ./build/tests/performance/olp-cpp-sdk-performance-tests --gtest_filter="*short_test_disk_cache" 2>> errors.txt || TEST_FAILURE=1
-mv heaptrack.olp-cpp-sdk-performance-tests.*.gz short_test_disk_cache.gz
+heaptrack ./build/tests/performance/olp-cpp-sdk-performance-tests --gtest_filter="*ReadNPartitionsFromVersionedLayer/short_test_null_cache" 2>> errors.txt || TEST_FAILURE=1
+mv heaptrack.olp-cpp-sdk-performance-tests.*.gz heaptrack.ReadNPartitionsFromVersionedLayer.short_test_null_cache.gz
+heaptrack ./build/tests/performance/olp-cpp-sdk-performance-tests --gtest_filter="*ReadNPartitionsFromVersionedLayer/short_test_memory_cache" 2>> errors.txt || TEST_FAILURE=1
+mv heaptrack.olp-cpp-sdk-performance-tests.*.gz heaptrack.ReadNPartitionsFromVersionedLayer.short_test_memory_cache.gz
+heaptrack ./build/tests/performance/olp-cpp-sdk-performance-tests --gtest_filter="*ReadNPartitionsFromVersionedLayer/short_test_disk_cache" 2>> errors.txt || TEST_FAILURE=1
+mv heaptrack.olp-cpp-sdk-performance-tests.*.gz heaptrack.ReadNPartitionsFromVersionedLayer.short_test_disk_cache.gz
+heaptrack ./build/tests/performance/olp-cpp-sdk-performance-tests --gtest_filter="*PrefetchPartitionsFromVersionedLayer/short_test_null_cache" 2>> errors.txt || TEST_FAILURE=1
+mv heaptrack.olp-cpp-sdk-performance-tests.*.gz heaptrack.PrefetchPartitionsFromVersionedLayer.short_test_null_cache.gz
+heaptrack ./build/tests/performance/olp-cpp-sdk-performance-tests --gtest_filter="*PrefetchPartitionsFromVersionedLayer/short_test_memory_cache" 2>> errors.txt || TEST_FAILURE=1
+mv heaptrack.olp-cpp-sdk-performance-tests.*.gz heaptrack.PrefetchPartitionsFromVersionedLayer.short_test_memory_cache.gz
+heaptrack ./build/tests/performance/olp-cpp-sdk-performance-tests --gtest_filter="*PrefetchPartitionsFromVersionedLayer/short_test_disk_cache" 2>> errors.txt || TEST_FAILURE=1
+mv heaptrack.olp-cpp-sdk-performance-tests.*.gz heaptrack.PrefetchPartitionsFromVersionedLayer.short_test_disk_cache.gz
```

Comment:
Would it make sense to separate these commands with empty lines to improve readability?

Reply:
Would it make sense to extract the common part into a function?

For example, the function could look as such:
```bash
function_name () {
mv heaptrack.olp-cpp-sdk-performance-tests.*.gz heaptrack.$1.$2.gz
heaptrack ./build/tests/performance/olp-cpp-sdk-performance-tests --gtest_filter="*$1/$2" 2>> errors.txt || TEST_FAILURE=1
}
```

Reply:
will fix with empty lines.